internal/web/templates/pages/package.html

```diff
@@ -97,6 +97,95 @@ <h2 class="text-reset">Package Dependencies</h2>
+                    id="pkg-install-version"
+                    name="selectedVersion"
+                    required
+                    hx-select="main"
+                    hx-swap="main"
+                    hx-target="main"
+                    hx-get="/packages/{{ .Manifest.Name }}">
+                    {{ with $idx := .PackageIndex }}
+                      {{ range $idx.Versions }}
+                        <option value="{{ .Version }}" {{ if eq .Version $.SelectedVersion }}selected{{ end }}>
+                          {{ .Version }}
+                          {{ if eq .Version $idx.LatestVersion }}(latest){{ end }}
+                        </option>
+                      {{ end }}
+                    {{ end }}
+                  </select>
+                </div>
+                <div class="form-check mt-1 mb-3">
+                  <input class="form-check-input" type="checkbox" name="enableAutoUpdate" id="enable-auto-update" />
+                  <label class="form-check-label ms-1" for="enable-auto-update">Auto Updates</label>
+                </div>
+              {{ end }}
+
+              {{ if ne (len .Manifest.ValueDefinitions) 0 }}
+                {{ if eq .Status nil }}
+                  <h4 class="text-reset">Configuration</h4>
+                {{ end }}
+                {{ range $valName, $valDef := .Manifest.ValueDefinitions }}
+                  {{ template "pkg-config-input" ForPkgConfigInput $.Package $.Manifest.Name $valName $valDef nil }}
+                {{ end }}
+              {{ end }}
+              {{ if $.ShowConflicts }}
+                <div class="alert alert-danger m-0 mb-2" role="alert">
+                  <span>Cannot install due to dependency conflicts:</span>
+                  <ul class="mb-0 mt-1">
+                    {{ range .ValidationResult.Conflicts }}
+                      <li>{{ .Actual.Name }} (required: {{ .Required.Version }}, actual: {{ .Actual.Version }})</li>
+                    {{ end }}
+                  </ul>
+                </div>
+              {{ else if eq .Status nil }}
+                <div class="alert alert-success m-0 mb-2" role="alert">
+                  <span>The following packages will be installed:</span>
+                  <ul class="mb-0 mt-1">
+                    <li>{{ $.Manifest.Name }} ({{ $.SelectedVersion }})</li>
+                    {{ range .ValidationResult.Requirements }}
+                      <li>{{ .Name }} ({{ .Version }})</li>
+                    {{ end }}
+                  </ul>
+                </div>
+              {{ end }}
+              <button
+                type="submit"
+                class="btn btn-primary btn-sm {{ if .ShowConflicts }}disabled{{ end }}"
+                {{ if .ShowConflicts }}disabled{{ end }}>
+                {{ if eq .Status nil }}
+                  Install
+                {{ else }}
+                  Save Configuration
+                {{ end }}
+              </button>
```

Comment:
> Finally, in case of conflicts, I think the button should be hidden altogether.

Not 100% convinced, because we do almost the opposite when uninstalling, except that the button is enabled, and the user can't continue on the modal then.